lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -213,8 +257,12 @@ sub _get_version_in_folder {
 # Obs version is parsed from files_iso.lst, which is updated from ObsRsync Gru tasks
 sub _get_obs_version {
     my ($c, $project) = @_;
-    my $home = $c->obs_rsync->home;
-    return _get_version_in_folder(Mojo::File->new($home, $project));
+    my $helper = $c->obs_rsync;
+    my $home   = $helper->home;
+    my $batch;
+    ($batch, $project) = $helper->get_first_batch($project);
```

Comment:
You could shorten this with `(my $batch, $project) = $helper->get_first_batch($project);`.